src/Duo.Clients.Wpf/Messaging/ApriManutenzioneTrattamentoMessage.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Duo.Domain.ViewModels.Trattamenti;
+
+namespace Duo.Clients.Wpf.Messaging
+{
+    class ApriManutenzioneTrattamentoMessage
+    {
+        public TrattamentoView Trattamento { get; set; }
```

Comment:
è un po' tirata per i capelli, stai partendo dall'assunzione che il modello che usi per visualizzare i risultati, in questo caso `TrattamentoView` sia anche quello che va bene per poi fare edit di un `Trattamento`. Potrebbe essere che vada benissimo, e probabilmente in questo caso va bene.